src/bot/ignoreList.ts

```diff
@@ -1,9 +1,14 @@
+type IsIgnoredArgs = {
+  account: string;
+  ignoreListVar: string;
+};
+
 /**
  * Check whether an account is in the ignore list
- * @param matrixAccount The matrix account
- * @returns boolean
  */
-export function isIgnored (matrixAccount: string): boolean {
-  const ignoreList: string[] = process.env.FAUCET_IGNORE_LIST?.split(',') ?? [];
-  return ignoreList.includes(matrixAccount);
+export function isIgnored ({
```

Comment:
Let's consider we have a big list of ignored accounts. Here you pass the huge list over and over again (for every faucet request). That does not sound optimum.

Reply:
You are right. So better to check it without extra functions.